plenum/config.py

```diff
@@ -393,6 +393,9 @@
 # Limit for numbers of 3pc and checkpoint messages stashed in replica
 REPLICA_STASH_LIMIT = 100000
 
+# Limit for number of messages that are allowed to be stashed in view change service
+VIEW_CHANGE_SERVICE_STASH_LIMIT = 1000
```

Comment:
Should we set it to 10.000? Our previous load tests showed that even 10,000 is not enough sometimes, so I suspect that 1000 may not be enough for badly lagging nodes

Reply:
I'm not sure. View change messages can be quite big and we don't expect a lot of them - for one view change on 25-nodes pool we don't expect more than 25 incoming messages on most nodes, or more than 49 incoming messages if it was primary per view change. So in order to hit limit of 1000 messages we'll need 20-40 consecutive view changes.

Reply:
ok, agree. I forgot that this is for ViewChange messages only, not for all 3PC messages for future view.